docs/orchestration/faq/upgrading_1.0.md

```diff
@@ -0,0 +1,165 @@
+| -w, --watch        | Wait for the flow run to finish executing and display status information. |
+
+For example, run a flow in a script locally:
+
+```bash
+$ prefect run -p hello-world.py
+```
+
+Run a flow with a non-default parameter locally:
+
+```bash
+$ prefect run -m prefect.hello_world --param name=Marvin
+```
+
+Run a registered flow with the backend with custom labels:
+
+```bash
+$ prefect run -n "hello-world" --label example --label hello
+```
+
+Run a registered flow and execute locally without an agent (for more information, see "Agentless execution" in the blog post [Prefect 0.15.0: A New Flow Run Experience](https://www.prefect.io/blog/prefect-0-15-0-a-new-flow-run-experience/)):
+
+```bash
+$ prefect run -n "hello-world" --execute
+```
+
+## Tasks for sub-flows
+
+Introduced in 0.15.0, Prefect includes new tasks that give you more flexibility around working with sub-flow execution &mdash; also known as "flow-of-flows" &mdash; and result passing:
+
+- [`create_flow_run`](/api/latest/tasks/prefect.html#create-flow-run) lets you programmatically create a flow run in the backend for a registered flow.
+- [`wait_for_flow_run`](h/api/latest/tasks/prefect.html#wait-for-flow-run) lets you wait for a flow run to finish executing while receiving state and log information regarding the flow run.
+- [`get_task_run_result`](/api/latest/tasks/prefect.html#get-task-run-result) waits for a task run to complete and returns the result.
+
+See [Scheduling a flow-of-flows](/core/idioms/flow-to-flow.html#scheduling-a-flow-of-flows) and the "Sub-flow result passing" section in the blog post [Prefect 0.15.0: A New Flow Run Experience](https://www.prefect.io/blog/prefect-0-15-0-a-new-flow-run-experience/) for details.
+
+## Imports have moved
+
+Imports for some Prefect modules have moved:
+
+- Artifacts functions now imported from `prefect.backend.artifacts`.
+- `Parameter` now imported from `prefect.Parameter` instead of `prefect.core.tasks`.
+- Exceptions now imported from `prefect.exceptions` instead of `prefect.utilities.exceptions`.
+- Executors now imported from `prefect.executors` instead of `prefect.engine.executors`. 
+
+## iCal recurrence rules schedules
+
+Prefect now supports rich recurrence rule scheduling following the iCal RRules standard and `dateutil` `rrule` module. This feature does not impact existing schedules using interval clocks, cron clocks, and so on, but provides convenient, new syntax for creating repetitive schedules. See [Recurrence Rule Clocks](/core/concepts/schedules.html#recurrence-rule-clocks) for details.
+
+This feature was contributed by a Prefect community member. To learn more, see the original pull request [Support for RRule (iCal style recurrence rule) clocks/schedules](https://github.com/PrefectHQ/prefect/pull/4901).
+
+## Drop support for Python 3.6
+
+With Prefect 1.0, we no longer explicitly test against Python 3.6. The minimum version for test and build is Python 3.7. 
```

Comment:
```suggestion
With Prefect 1.0, we no longer support Python 3.6. Some features will not work as expected if you are using Python 3.6. The minimum recommended version is Python 3.7. 
```

Not only do we not test against 3.6, some features will break with 3.6. I don't think we need to soften this.

Reply:
+1